tuhi/wacom.py

```diff
@@ -99,6 +99,11 @@ class WacomProtocol(GObject.Object):
     '''
     Internal class to handle the communication with the Wacom device.
 
+    This is the top-level class and should represent the latest device's
+    communication patterns. Any device-specific quirks are handled in the
+    subclasses of this device, reverse-ordered by date. Right now, this
+    means WacomProtocol represents the IntuosPro, subclassing that is the
+    Slate which itself is subclassed by the Spark.
```

Comment:
After a little bit of thinking, I think we should do this the other way around.
Having the older first and the newer inherits from their parents.

The reason is that with a reverse ordering, we will have to check on every generation when adding a new device. While if we have it the older first, adding a new device protocol will not require us to change anything in the older devices, and so we will not have to test those but the new ones.